lib/delegate.js

```diff
@@ -277,6 +278,11 @@ Delegate.prototype.handle = function(event) {
     target = target.parentNode;
   }
 
+  // Handle SVG <use> elements in IE
+  if (target.correspondingUseElement) {
+    target = target.correspondingUseElement;
```

Comment:
To reproduce the error in IE:
- Comment this line
- Comment `delegate.off();` in the new test case
- Run tests via `./node_modules/.bin/grunt && ./node_modules/.bin/buster-static -c test/buster.js`
- Hover a blue circle in IE

Result: https://www.dropbox.com/s/upmu63lko9986ys/Screenshot%202019-07-09%2021.13.01.png?dl=0